cdp_scrapers/instances/houston.py

```diff
@@ -0,0 +1,531 @@
+from bs4 import BeautifulSoup, Tag, NavigableString
+import requests
+from cdp_backend.pipeline import ingestion_models
+from datetime import datetime, timedelta
+from typing import List, Union
+import bs4
+
+# from typing import Tuple, Optional
+
+
+# def get_role(name: str) -> Tuple[List[str], bool]:
```

Comment:
leave a comment that says: "Houston has no person info currently but these functions are ready whenever we do have person data"